### PR TITLE
[WIP] Reswizzle getting started tutorials

### PR DESCRIPTION
**Pull request by everett-toews**

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-586bfd7137/)

**Comment by carolynvs** [comment]
Quick question about the shared laptops and the tutorial we point them to. They will all have carina and dvm installed, and some people found it confusing. Does it make sense to have a version of the getting started guides which are just for the booth experience?

**Comment by everett-toews** [comment]
Maybe? I really want to minimize the amount extra special stuff we have to do in order to support events. Gonna have to give that some more thought.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-9c8660fe68/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/getcarina.com/build-d74fc9f4ed/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-62c00673f9/)

File: _includes/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+    On Windows PowerShell, run the following command:
+
+    ```
+    > iex (wget https://download.getcarina.com/dvm/latest/install.ps1)
+    ```
+
+1. Copy the commands from the output, and then paste and run them to finalize the installation.
+
+1. Configure the Docker client.
```
Comment (carolynvs) [inline _includes/getting-started.md]: Are we showing both bash/powershell just to be consistent? The command is the some in both cases.

Comment (everett-toews) [inline _includes/getting-started.md]: Ya. I noticed that too. The only difference being the `$` and the `>`. I left them both in for consistency. It'd be nice to only have one though.

If there was only one, I guess we'd use `$`. Do you think that would that be too confusing for Windows users?

**Comment by carolynvs** [comment]
Looks good! Just ran through this on powershell. My only suggestion is to consolidate the `dvm use` step on both tutorials. :shipit:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-0793bf97a0/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-bba6a97dda/)

**Pull request merged**